normcap/screengrab/handlers/dbus_portal.py

```diff
@@ -6,7 +6,7 @@
 import sys
 from pathlib import Path
 from typing import Optional
-from urllib.parse import urlparse
+from urllib.parse import urlparse, unquote
```

Comment:
```suggestion
from urllib.parse import unquote, urlparse
```

imports should be sorted